packages/cli/src/commands/setup/auth/__tests__/__snapshots__/addAuthConfigToApp.test.js.snap

```diff
@@ -67,13 +67,19 @@ const firebaseClientConfig = {
   appId: process.env.FIREBASE_APP_ID,
 }
 
-const firebaseClient = ((config) => {
-  if (!firebase.apps.length) {
-    firebase.initializeApp(config)
+const firebaseApp = ((config) => {
```

Comment:
I think these variable names here are confusing in the template.

Should we rename `firebaseClientConfig` to `firebaseConfig` so its not weird that we're passing client config to the app?